docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -187,6 +187,9 @@ end
   to understand; it is also more difficult to see all the code at once. Consider
   splitting the function into multiple ones, if it is sensibly possible.
 
+- Every export statement must export exactly one identifier. If multiple
+  export statements appear in sequence, they must be sorted alphabetically.
```

Comment:
I think it's a good idea to explain the *why* behind some of our rules. If we can't explain a rule, then it's probably not a good rule. Knowing the motivation for a rule can also make it easier to accept it psychologically. Plus it helps to decide when it might be sensible to break a rule, or to revise it. 
```suggestion
  export statements appear in sequence, they must be sorted alphabetically.
  The purpose of this is to make it easy to decide where to put additional
  exports, and also to detect duplicate exports (which are not harmful by
  themselves, but if we decide to no longer export an identifier, having
  multiple exports can lead to mistakes such as removing only one of them.
```